book-edited.tex

```diff
@@ -19516,7 +19516,7 @@ \section{Overlap and Transforms}\label{overlap-and-transforms}}
 
 However, if you try to load the example above, you'll find that it still
 looks wrong---the blue square is supposed to be \emph{under} the green
-one, but it's on top.\footnote{Hit-testing is correct, though, because
+one, but it's on top.\footnote{Hit testing is correct, though, because
```

Comment:
It seems we settled on "hit testing", no "hit-testing", though the copy editor seems
inconsistent about it.

Reply:
Can we check if we really are consistent? No hyphen seems right, we are testing for a hit.

Reply:
I checked.